curiouslearning/CRcontainer
# CR-108 pulse animation added

# Changes
- WebAppsAdapter.java
- app/src/main/res/layout/activity_custom_list.xml
- app/build.gradle
- app/src/main/res/anim/pulse_scale.xml
- app/src/main/res/drawable/pulse_animation.xml

# How to test
- Check pulse animation for FTM subapp icon.
- When I tap on the FTM icon launch and later return to the Curious Reader homescreen OR close the Curious Reader app and open it again to also be on the homescreen,

Ref: [CR-108](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-108)


[CR-108]: https://curiouslearning.atlassian.net/browse/CR-108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
